content/applications/inventory_and_mrp/manufacturing/management/quality_control.rst

```diff
@@ -0,0 +1,80 @@
+================================
+Configure quality control checks
+================================
+
+Quality control is an essential part of the manufacturing process. Odoo's Quality module can be
+used to control the quality of products before they are registered into stock, during picking
+operations, and when leaving the warehouse in a delivery order. This is accomplished by creating
+*Quality Control Points* that will automatically trigger quality checks at specific points during
+production.
+
+Create and configure quality control points
+===========================================
+
+To create a quality control point, go to :menuselection:`Quality Control --> Control Points -->
+Create`. From here, configure the new control point based on the desired quality check workflow.
+
+.. image:: quality_control/quality-control-form.png
+   :align: center
+   :alt: Configure a quality control point in Odoo.
+
+Products
+--------
+
+Select the specific product or products that the check is performed on.
+
+Operations
+----------
+
+Choose the type of operation that should be performed during the quality check. Certain operations
+also allow for the selection of a specific work order.
+
+.. example::
+   Specify that a quality check should be performed during the manual assembly work order of the
+   manufacturing operation.
+
+.. tip::
+   To perform a quality check on products produced by subcontractors, select :guilabel:`Your
+   Company: Receipts`. A quality check will be created each time delivery of a product is
+   validated.
+
+Control settings
+----------------
```

Comment:
This section's good, but it took too long to get to this here and dig deeper on the product. Would move up after the overview list I made at the top.